src/metabase/db/liquibase.clj

```diff
@@ -127,11 +123,11 @@
       (log/info
         (trs "Migration lock cleared, but nothing to do here! Migrations were finished by another instance.")))))
 
-(defn force-migrate-up-if-needed!
-  "Force migrating up. This does two things differently from `migrate-up-if-needed!`:
+(s/defn force-migrate-up-if-needed!
+  "Force migrating up. This does two things differently from [[migrate-up-if-needed!]]:
```

Comment:
Should this comment now say "three things"?

Reply:
fixed